sceptre/scipool/config/develop/sc-product-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -1,114 +0,0 @@
-template:
```

Comment:
i would recommend keeping this until everything is working.  remove all of these in one PR.